File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/GetUserPayloadResponse.java
package com.lattis.ellipse.data.network.model.response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by ssd3 on 5/4/17.
 */

public class GetUserPayloadResponse {

    @SerializedName("user")
    UserResponse userResponse;

    @SerializedName("private_account")
    List<PrivateNetworkResponse> privateNetworksResponse;

    public List<PrivateNetworkResponse> getPrivateNetworkResponse() {
        return privateNetworksResponse;
    }

    public UserResponse getUserResponse() {
        return userResponse;
    }


}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/user/GetUserPayloadResponse.kt
package com.lattis.data.entity.response.user

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.private_networks.PrivateNetworkResponse
import com.lattis.data.entity.response.user.UserResponse


class GetUserPayloadResponse {

    @SerializedName("user")
    var userResponse: UserResponse? = null
    @SerializedName("private_account")
    var privateNetworkResponse: List<PrivateNetworkResponse>? = null
}
